metrics_utility/test/ccspv_reports/test_CCSPv2.py

```diff
@@ -0,0 +1,111 @@
+import pandas as pd
+import os
+import subprocess
+import sys
+
+env_vars = {
+    "METRICS_UTILITY_PRICE_PER_NODE": "11.55",
+    "METRICS_UTILITY_REPORT_RHN_LOGIN": "test_login",
+    "METRICS_UTILITY_SHIP_PATH": "/awx_devel/awx-dev/metrics-utility/metrics_utility/test/test_data",
+    "METRICS_UTILITY_REPORT_END_USER_COMPANY_NAME": "Customer A",
+    "METRICS_UTILITY_REPORT_END_USER_STATE": "TX",
+    "METRICS_UTILITY_REPORT_SKU_DESCRIPTION": "EX: Red Hat Ansible Automation Platform, Full Support (1 Managed Node, Dedicated, Monthly)",
+    "METRICS_UTILITY_REPORT_H1_HEADING": "CCSP NA Direct Reporting Template",
+    "METRICS_UTILITY_REPORT_END_USER_CITY": "Springfield",
+    "METRICS_UTILITY_REPORT_PO_NUMBER": "123",
+    "METRICS_UTILITY_SHIP_TARGET": "directory",
+    "METRICS_UTILITY_REPORT_END_USER_COUNTRY": "US",
+    "METRICS_UTILITY_REPORT_COMPANY_NAME": "Partner A",
+    "METRICS_UTILITY_REPORT_SKU": "MCT3752MO",
+    "METRICS_UTILITY_REPORT_EMAIL": "email@email.com",
+    "METRICS_UTILITY_REPORT_TYPE": "CCSPv2",
+    "AWX_LOGGING_MODE": "stdout",
+}
+
+file_path = (
+    "/awx_devel/awx-dev/metrics-utility/metrics_utility/test/test_data/reports/2024/02/CCSPv2-2024-02.xlsx"
+)
+
+
+EXPECTED_SHEETS = {
+    "Usage Reporting": [
+        "CCSP NA Direct Reporting Template",
+        "Unnamed: 1",
+        "Unnamed: 2",
+        "Unnamed: 3",
+        "Unnamed: 4",
+        "Unnamed: 5",
+        "Unnamed: 6",
+        "Updated: Jan 21, 2025",
```

Comment:
This one must be changed. Updated should match current date, it should not be fixed. Today is for example Jan 22, 2025 and test fails :-)